modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestResponse.siddhi

```diff
@@ -0,0 +1,77 @@
+@App:name("PublishHttpOAuthRequestResponse")
+@App:description("Send and get recive an OAuth events via HTTP transport")
```

Comment:
This description has to be more descriptive and accurate; so that the user will be able to understand the purpose of this siddhi app. 

Here we are sending an HTTP message via an OAuth protected endpoint and catching its response.